prov/ansible/monitoring_job.go

```diff
@@ -78,6 +80,7 @@ func (o *actionOperator) ExecAction(ctx context.Context, cfg config.Configuratio
 			if opErr == nil {
 				opErr = errors.Errorf("job implementation of node %q was detected as failed", nodeName)
 			}
+			deployments.SetInstanceStateStringWithContextualLogs(ctx, consulutil.GetKV(), deploymentID, action.Data["nodeName"], "0", tosca.NodeStateError.String())
```

Comment:
No need to call deployments.SetInstanceStateStringWithContextualLogs() when the state is completed ?

Reply:
Yes, I think you're right

Reply:
But which state ? OK or "completed" ?

Reply:
It seems for Slurm Jobs, this is `Completed`, while for kubernetes jobs this is `Succeeded` and the failure is `Failed`

Reply:
Ok it seems to be the specific states for job schedulers. But in our case, there is no job scheduler and we should just follow Tosca nodes states ? WYT ?

Reply:
ok, started then ?

Reply:
Yes

Reply:
Done